jax/random.py

```diff
@@ -57,11 +58,42 @@
     >>> random.uniform(subkey)
     Array(0.10536897, dtype=float32)
 
+.. note::
+
+   Typed key arrays, with element types such as `key<fry>` above, were
```

Comment:
In rst, use double back-ticks for fixed-width (single back-ticks are rendered as itallic)

Also several instances below.